README.md

```diff
@@ -2,8 +2,69 @@
 Welcome!
 
 ---
-##### Third Party Notice
+## Third Party Notice
 This repository is not responsible for scripts malfunctioning due to issues that are not caused by asl-help.
 We reserve the right to immediately close any issues which report problems out of our control.
 
 Instead, please either open an issue in the repository of the script's creator, or ask a question in the [Speedrun Tool Development Discord server](https://discord.gg/cpYsxz7).
```

Comment:
Please move this to the very bottom and retain the horizontal rule and H5.

Reply:
Markdown only has three heading levels.

Reply:
Nevermind it has six??? I never noticed.